packages/react-examples/src/react/OverflowSet/OverflowSet.Vertical.Example.tsx

```diff
@@ -14,25 +15,28 @@ const onRenderOverflowButtonStyles = {
 
 const onRenderItem = (item: IOverflowSetItemProps): JSX.Element => {
   return (
-    <CommandBarButton
-      role="menuitem"
-      aria-label={item.name}
-      styles={onRenderItemStyles}
-      iconProps={{ iconName: item.icon }}
-      onClick={item.onClick}
-    />
+    <TooltipHost content={item.title} id={`menuitem-${item.key}-tooltip`}>
+      <CommandBarButton
+        role="menuitem"
+        aria-label={item.name}
+        styles={onRenderItemStyles}
+        iconProps={{ iconName: item.icon }}
+        onClick={item.onClick}
+      />
+    </TooltipHost>
   );
 };
 
 const onRenderOverflowButton = (overflowItems: any[] | undefined): JSX.Element => {
   return (
-    <CommandBarButton
-      role="menuitem"
-      title="More items"
-      styles={onRenderOverflowButtonStyles}
-      menuIconProps={{ iconName: 'More' }}
-      menuProps={{ items: overflowItems! }}
-    />
+    <TooltipHost content="More items" id="menuitem-more-tooltip">
+      <CommandBarButton
+        role="menuitem"
+        styles={onRenderOverflowButtonStyles}
```

Comment:
You should add `aria-label="More items"` here (otherwise I don't think it will be read correctly by screen readers)